data/cpu_bugs/vm_install_script/sle-15/create_vm_url.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+if [ $# -lt 4 ]; then
+    echo "$0 <name> <install_url> <autoyast_url> <logfile_path> <vm_pool> [cpu]"
```

Comment:
The check for the number of parameters is not matched with its help message.

Reply:
Yes, It's a mistake, it should be <5.